src/features/pastelDB/sql-wasm.js

```diff
@@ -0,0 +1,3094 @@
+// We are modularizing this manually because the current modularize setting in Emscripten has some issues:
```

Comment:
This file and the below `wasm` file are like binaries not code, do you think it's better to put it in `src/assets/wasm/` or something?

Reply:
Fixed this on a separate branch for @Dinisueldanov to copy changes from: https://github.com/pastelnetwork/pastel-electron-wallet/blob/eb085168be7ee800fc99e83993dc13cd97e66f8e/package.json#L68